### DISCUSSION TITLE
TextLayer size changed between 8.4 and 8.5

### DISCUSSION THREAD

**Discussion by jonenst** [visgl/deck.gl#6176] [General]
Hi, upgrading to deckgl 8.5, I noticed that my texts have gotten a lot bigger. Reproducing using the code from https://deck.gl/gallery/text-layer . Here's the result with deckgl 8.4 and 8.5:
deckgl8.5: https://codepen.io/jonenst/pen/XWgpegm
deckgl8.4.: https://codepen.io/jonenst/pen/qBjRPXR

Is this change intentional ? I didn't find it in the migration guide.
What is the recommended way to get the same results as before ? Thanks

To get an almost equivalent result to 8.4, I found that dividing the size by 1.5 did the trick (using rational size, e.g. 16 -> 10.6666..). 
deckgl8.5 with 1.5 division: https://codepen.io/jonenst/pen/MWoJEBw

screenshot
![Capture d’écran de 2021-09-08 11-39-08](https://user-images.githubusercontent.com/89208/132485945-5bb786d1-70ff-4046-9627-8d92646ee77b.png)

**Comment by jonenst** [comment]
This also affects getPixelOffset, so my texts are also unaligned compared to 8.4 which is very ugly. Is it a bug ?

**Reply by Pessimistress** [reply]
https://deck.gl/docs/upgrade-guide#upgrading-from-deckgl-v84-to-v85

**Reply by jonenst** [reply]
Hmm, when I read that, I didn't understand that the very first bullet point was talking about my problem. Maybe it need to be rephrased to be more understandable ? "A bug is fixed in projecting sizes in billboard mode. Pixel sizes now match their CSS counterparts. As a result, some items are 1.5 times bigger than they used to be in many cases, because previously you didn't get the sizes you were asking for."

Also the sentence "After upgrading to v8.5, in order to maintain the same appearance, you need to multiply 2/3 to the objects' width/size. This can be done by either changing the accessor (getWidth/getSize) or the scaling prop (sizeScale/widthScale)." should be moved out of the last bullet point because it affects all the layers (actually I think the first time I read the upgrade docs, I skimmed past this sentence because I thought it applied only to PathLayer, and my problem was with TextLayer. This is probably the reason why I didn't understand this section of the upgrade guide).

Thanks for your time and sorry for misunderstanding the upgrade guide in the first place

**Reply by Pessimistress** [reply]
Thank you for the feedback. Would you open a PR to apply these changes?

**Reply by jonenst** [reply]
Like this ? https://github.com/visgl/deck.gl/pull/6180
Cheers
